docs/source/fx.rst

```diff
@@ -12,17 +12,31 @@ Overview
 Writing Transformations
 -----------------------
 
-What is a FX transform? Essentially, it's a function that looks like this.
+What is an FX transform? Essentially, it's a function that looks like this.
 
 ::
 
-    def transform(m: nn.Module) -> nn.Module:
-        fx_model: GraphModule = fx.symbolice_trace(m)
-        new_model = ...
-        return new_model
+    import torch
+    import torch.fx
+
+    def transform(m: nn.Module,
+                  tracer_class : type = torch.fx.Tracer) -> torch.nn.Module:
+        # Step 1: Acquire a graph representing the code in `m`
+
+        # NOTE: torch.fx.symbolic_trace is a wrapper around a call to
+        # fx.Tracer.trace and constructing a GraphModule. We'll
+        # split that out in our transform to allow the caller to
+        # customize tracing behavior.
+        graph : torch.fx.Graph = tracer_class().trace(m)
 
-Your transform will take in an :class:`torch.nn.Module`, convert it into a
-:class:`GraphModule` with :meth:``symbolic_trace``, and return a new
+        # Step 2: Modify this graph or create a new one
+        graph = ...
+
+        # Step 3: Construct a Module to return
+        return torch.fx.GraphModule(m, graph)
+
+Your transform will take in an :class:`torch.nn.Module`, acquire a :class:`Graph`
+from it, do some modifications, and return a new
 ``nn.Module``. You should think of the ``nn.Module`` that your FX transform
 returns as identical to a regular ``nn.Module`` -- you can pass it to another
 FX transform, you can pass it to TorchScript, or you can
```

Comment:
grammar fix: Ensuring that the inputs and outputs of your FX transform are **an** ``nn.Module``